### PR TITLE
[#2706] - redesign datasets header (and entity type header)

### PR DESCRIPTION
**Pull request by marvinkome**
- [ ] **Update release notes if necessary**

**Screenshot**
image

- Change the dataset header to match the new design for handling question group. Note, this change affects both the visualization and other pages that uses the Header component or the EntityTypeHeader

- See also: #2706

### CODE REVIEW

File: client/src/translations/en.json [hunk 1]
```diff
@@ -261,6 +261,7 @@
   "reselect_source_column": "Pick another column",
   "resources": "Resources",
   "request_permission_from_your_organisation_admin": "Request permission from your organisation admin.",
+  "retry": "Retry",
```
Comment (tangrammer) [inline client/src/translations/en.json:264]: es.json version should be
"retry": "Volver a intentar"
"visualise": "Visualizar"


for fr.json maybe @loicsans can help :)

**Comment by marvinkome** [comment]
@valllllll2000 can you help with the French translations?

File: client/src/translations/en.json (hunk 1 above)
Comment (valllllll2000) [reply client/src/translations/en.json:264]: will do the french... give me a few minutes...

File: client/src/translations/es.json [hunk 2]
```diff
@@ -127,8 +127,8 @@
   "filter_visualisations_by_dataset": "Filtrar visualizaciones por conjunto de datos",
   "flow_application_url": "URL de la aplicación Flow",
   "first_name": "Nombre",
-  "following_visualisation_will_be_deleted":"Las siguiente visualización será borrada",
-  "following_visualisations_will_be_deleted":"Las siguientes visualizaciones serán borradas",
+  "following_visualisation_will_be_deleted": "Las siguiente visualización será borrada",
```
Comment (valllllll2000) [inline client/src/translations/es.json:130]:
```suggestion
  "following_visualisation_will_be_deleted": "La siguiente visualización será borrada",
```
I have seen a little mistake here: there should be no plural

File: client/src/components/common/Header.scss [hunk 3]
```diff
@@ -83,9 +82,9 @@
         margin-left: 20px;
     }
     
-    .actions {
-        border-left: 0.1rem solid $lightBlueGrey;
-    }
+    // .actions {
```
Comment (valllllll2000) [inline client/src/components/common/Header.scss:85]: Should we delete this commented out code or is it still needed?

**Comment by valllllll2000** [comment]
> @valllllll2000 can you help with the French translations?

French translations added

**Pull request merged**